src/Entity/AbstractEntity.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Silktide\ProspectClient\Entity;
+
+abstract class AbstractEntity
+{
+    /** @var mixed */
+    protected $jsonData;
```

Comment:
Once upon a time it was json data, but now it can be known by "data" :D

Reply:
:+1: Agreed.